content/zh/blog/_posts/2020-12-02-dont-panic-kubernetes-and-docker.md

```diff
@@ -0,0 +1,210 @@
+[buildah](https://github.com/containers/buildah). 
+-->
+要注意一点：如果你依赖底层的 docker 套接字(`/var/run/docker.sock`)，作为你集群中工作流的一部分，
+切换到不同的运行时会导致你无法使用它。
+这种模式经常被称之为嵌套 Docker（Docker in Docker）。
+对于这种特殊的场景，有很多选项，比如：
+[kaniko](https://github.com/GoogleContainerTools/kaniko)、
+[img](https://github.com/genuinetools/img)、和
+[buildah](https://github.com/containers/buildah)。
+
+<!-- 
+## What does this change mean for developers, though? Do we still write Dockerfiles? Do we still build things with Docker?
+-->
+## 那么，这一改变对开发人员意味着什么？我们还要写 Dockerfile 吗？还能用 Docker 构建镜像吗？{#what-does-this-change-mean-for-developers}
+
+<!-- 
+This change addresses a different environment than most folks use to interact
+with Docker. The Docker installation you’re using in development is unrelated to
+the Docker runtime inside your Kubernetes cluster. It’s confusing, we understand.
+As a developer, Docker is still useful to you in all the ways it was before this
+change was announced. The image that Docker produces isn’t really a
+Docker-specific image&mdash;it’s an OCI ([Open Container Initiative](https://opencontainers.org/)) image. 
+Any OCI-compliant image, regardless of the tool you use to build it, will look
+the same to Kubernetes. Both [containerd](https://containerd.io/) and
+[CRI-O](https://cri-o.io/) know how to pull those images and run them. This is
+why we have a standard for what containers should look like.
+-->
+此次改变带来了一个不同的环境，这不同于我们常用的 Docker 交互方式。
+你在开发环境中用的 Docker 和你 Kubernetes 集群中的 Docker 运行时无关。
+我们知道这听起来让人困惑。
+对于开发人员，Docker 从所有角度来看仍然有用，就跟这次改变之前一样。
+Docker 构建的镜像并不是 Docker 特有的镜像——它是一个
+OCI（[开放容器标准](https://opencontainers.org/)）镜像。
+任一 OCI 兼容的镜像，不管它是用什么工具构建的，在 Kubernetes 的角度来看都是一样的。
+[containerd](https://containerd.io/) 和
+[CRI-O](https://cri-o.io/)
+两者都知道怎么拉取并运行这些镜像。
+这就是我们制定容器标准的原因。
+
+<!-- 
+So, this change is coming. It’s going to cause issues for some, but it isn’t
+catastrophic, and generally it’s a good thing. Depending on how you interact
+with Kubernetes, this could mean nothing to you, or it could mean a bit of work.
+In the long run, it’s going to make things easier. If this is still confusing
+for you, that’s okay&mdash;there’s a lot going on here; Kubernetes has a lot of
+moving parts, and nobody is an expert in 100% of it. We encourage any and all
+questions regardless of experience level or complexity! Our goal is to make sure
+everyone is educated as much as possible on the upcoming changes. We hope
+this has answered most of your questions and soothed some anxieties! ❤️
+-->
+所以，改变已经发生。
+它确实带来了一些问题，但这不是一个灾难，总的说来，这还是一件好事。
+根据你操作 Kubernetes 的方式的不同，这可能对你不构成任何问题，或者也只是意味着一点点的工作量。
+从一个长远的角度看，它使得事情更简单。
+如果你还在困惑，也没问题——这里还有很多事情；
+Kubernetes 有很多变化中的功能，没有人是100%的专家。
+我们鼓励你提出任何问题，无论水平高低、问题难易。
+我们的目标是确保所有人都能在即将到来的改变中获得足够的了解。
+我们希望这已经回答了你的大部分问题，并缓解了一些焦虑。❤️
+
+<!-- 
+Looking for more answers? Check out our accompanying [Dockershim Deprecation FAQ](/blog/2020/12/02/dockershim-faq/).
+-->
+还在寻求更多答案吗？请参考我们附带的
+[弃用 Dockershim 的常见问题](/blog/2020/12/02/dockershim-faq/).
```

Comment:
There's already a localized version available, so you could:
```suggestion
[弃用 Dockershim 的常见问题](/zh/blog/2020/12/02/dockershim-faq/).
```

Reply:
wow, this piece is also my translation , corrected.